node/resources/addTranslations.ts

```diff
@@ -0,0 +1,49 @@
+import { SaveArgs as TranslateMessageArgs } from '@vtex/api'
+import { evolve } from 'ramda'
+import { Entity, FailedTranslation } from '../typings/typings'
+
+type MessagesOfProvider = TranslateMessageArgs['messagesByProvider'][0]
+
+function getEntityProvider(entity: Entity) {
```

Comment:
:pick: This could be an object.
```ts
const providerByEntity: Record<Entity, string> = {
  brand: 'Brand-id',
  category: 'Category-id',
  product: 'Product-id',
  sku: 'SKU-id',
  specification: 'Specification-id',
}
```